portfolio/src/main/webapp/index.html

```diff
@@ -12,23 +12,153 @@
   </head>
   <body>
 
-    <div id="header"> 
+    <div class="loader" id="loaderid"></div>
+
+    <div class="hiddenclass" id="header"> 
         <div id="header-spacer"></div>
         <img class="headerels headerimages" src = "images/google.png"/>
-        <p class="headerels">Seun Omonije STEP Portfolio</p>
-        <input id="searchbar" class="headerels" type="text" placeholder="Search about me! (unimplemented)" />
+        <p class="headerels">STEP Portfolio</p>
+        <input id="searchbar" class="headerels" type="text" placeholder="Learn about me! (try: my name, projects, who I am) Press enter to search" />
+
+        <!-- For some reason i need this in, don't know why-->
+        <input name="buttonExecute" style="display:none" onclick="runSearch(document.getElementById('searchbar').value.toLocaleLowerCase())" type="button" value="Search" />
     </div>
 
+    <div id="contentspacer"></div>
+
+    <div class = "hiddenclass" id="content">
+        
+        <div id="welcome-wrapper">
```

Comment:
This is a good use of whitespace in that you separated this "welcome-wrapper" section from the "part" section below, both of which are logical, large chunks of code, so adding the space improves the readability of the file. 

In general, try to save whitespace (blank lines) for when you're separating these types of logical chunks of code. Too much whitespace means those chunks don't stand out anymore, so the readability benefit is lost (it also leads to really long files!). As a reviewer, I want to be able to scan through the template and easily notice the different features on the page!